web-shop-service/src/main/java/io/smartcat/avro/events/item/ItemEvent.java

```diff
@@ -0,0 +1,450 @@
+/**
```

Comment:
Add this auto generated files to gitignore. Do not commit them.

Reply:
I pushed it on purpose because I was lazy to automate registering schemas to a schema registry. But I will :)